### PR TITLE
DM-3507 Documentation & Change Log for new MQTT static template

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
Small documentation addition.

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-20-325-0-New-Static-Template-150-CRA.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date:
+title: Set cloud remode access protocols using SmartREST
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-325-0-New-Static-Template-150-CRA.md:3]:
```suggestion
title: Set supported Cloud Remote Access protocols using SmartREST
```

File: content/change-logs/device-management/cumulocity-10-20-325-0-New-Static-Template-150-CRA.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-LcWEQW5gs
+    label: MQTT
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3507
+version: 10.20.325.0
+---
+A new SmartREST template has been added. Using the template id 150, it is now possible to set the list of cloud remote access protocols supported by a device.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-325-0-New-Static-Template-150-CRA.md:17]:
```suggestion
A new SmartREST template has been added. Using the template ID 150, it is now possible to set the list of Cloud Remote Access protocols supported by a device.
```

File: content/smartrest/mqtt-static-templates.md [hunk 3]
```diff
@@ -563,6 +564,20 @@ Set the supported software types of the device. Ignores empty elements. An empty
 143,yum,docker
 ```
 
+##### Set supported software types (150) {#150}
+
+Sets the list of cloud remote access protocols supported by the device. Possible values are `SSH`,`TELNET`,`VNC` and `PASSTHROUGH`. Empty elements are ignored. An empty list removes the `c8y_RemoteAccessSupportedProtocols` fragment entirely.
```
Comment (BeateRixen) [inline content/smartrest/mqtt-static-templates.md:569]:
```suggestion
Sets the list of Cloud Remote Access protocols supported by the device. Possible values are `SSH`,`TELNET`,`VNC` and `PASSTHROUGH`. Empty elements are ignored. An empty list removes the `c8y_RemoteAccessSupportedProtocols` fragment entirely.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
Since this PR also holds the documentation change (in addition to the change log entry) we should give it a "not-yet-deployed" label and only merge it with the next "cumulocity" component deployment to eu-latest. Otherwise the documentation change is immediately live.

**Review by aaronraab — APPROVED** [review]

**Review by l2p- — APPROVED** [review]

**Pull request merged**